plugins/azure/sqldatabases/dbDiagnosticLoggingEnabled.js

```diff
@@ -0,0 +1,100 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Database Diagnostic Logging Enabled',
+    category: 'SQL Databases',
+    domain: 'Databases',
+    description: 'Ensures diagnostic logging is enabled for SQL databases.',
+    more_info: 'Enabling diagnostic logging provides valuable insights into SQL database that helps to monitor resources for their availability, performance, and operation.',
+    recommended_action: 'Enable diagnostic logging for SQL databases with the minimum required data recording settings: SQLInsights, Errors, Timeouts, Blocks, Deadlocks, Basic, InstanceAndAppAdvanced, WorkloadManagement.',
+    link: 'https://learn.microsoft.com/en-us/azure/azure-sql/database/monitoring-sql-database-azure-monitor?view=azuresql',
+    apis: ['servers:listSql', 'databases:listByServer', 'diagnosticSettings:listByDatabase'],
+    realtime_triggers: ['microsoftsql:servers:write', 'microsoftsql:servers:delete', 'microsoftsql:servers:databases:write', 'microsoftsql:servers:databases:delete','microsoftinsights:diagnosticsettings:write','microsoftinsights:diagnosticsettings:delete'],
+    settings: {
+        diagnostic_logs: {
```

Comment:
```suggestion
        sqldb_diagnostic_logs: {
```
should make sure that this setting name is unique in all diagnostic log plugins